src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,293 @@
+
+* Alena Osipova
+* Artur Smęt
+* Daniela
+* Dilyan Grigorov
+* Dmitry Osipov
+* Doreen Peace Nangira Wanyama
+* Emma Saroyan
+* Emmanuel Akanji
+* Foram
+* Gajendraeshpande
+* George Zisopoulos
+* Goetz
+* Harshit Gupta
+* Hitesh Khandelwal
+* Isaac Chung
+* Jahanvi Gupta
+* Jaz Allibhai
+* Jiawei Wang
+* Julian maurin
+* Kalyan Prasad
+* Kevin Romero
+* Konstantin Ignatov
+* Koti Vellanki
+* Krystian
+* Kshiteej
+* Lajoie Batadiwa
+* Leila Ben Othman
+* Manav Mehra
+* Mayank Khanduja
+* Naa Ashiorkor Nortey
+* Nancy Henrietta Addo
+* Neeraj Pandey
+* Oliver Thieke
+* Osagie Eboigbe
+* Padraic Calpin
+* Patrick Arminio
+* Pavel Bityukov
+* Priya Basker
+* Ria Bhatia
+* Rodrigo Girão Serrão
+* Roland Sztaho
+* Sameer Shukla
+* Sarah Abderemane
+* Saurav Jain
+* Saurav Pandey
+* Sebastian M. Ernst
+* Siddharth Gupta
+* Simon Peteragbui
+* Tim
+* Tim Shchetilin
+* Vamsi Krishna Kolli
+* Vinícius Gubiani Ferreira
+* Wachira Maina
+* William
+* kt zhao
+* masamori0083
+* Shekhar Koirala
+
+
+# Speaker Mentorship Programme
+
+This team helped and supported potential speakers who want help preparing for their talk at EuroPython 2024.
+
+- Marijka Romaniuk
+- Naa Ashiorkor Nortey
+- Valery Calderón Briz
+- Theofanis Petkos
+
+### Mentors
+
+Here are all the mentors who lent their time and expertise to new speakers.
+
+- Alena Osipova
+- Artur Smęt
+- Ashmi Banerjee
+- Bernice Waweru
+- Bhagvan Kommadi
+- Dilyan Grigorov
+- Gajendra Deshpande
+- Honza Kral
+- Jean-Baptiste Braun
+- Jiawei Wang
+- Kalyan Prasad
+- Kenni Konate
+- Koti Vellanki
+- Lina Marieth Hoyos
+- Mario García
+- Max Kahan
+- Maxim Danilov
+- Nabanita Roy
+- Neeraj Pandey
+- Neil Vaytet
+- Priyanshu Krishnan
+- Reuven Lerner
+- Ria Bhatia
+- Sameer Shukla
+- Sebastian Witowski
+- Siddharth Gupta
+- Valery Calderón Briz
+- Vinícius Gubiani Ferreira
+- Yuliia Barabash
+
+
+# Operations Team
+
+From managing recording & streaming, to liaising with our wonderful remote and on-site volunteers, the Operations team have all covered. If you ever received a reply from our helpdesk, chances are you interacted with someone from this team :)
+
+- Anežka Müller
+- Angel Ramboi
+- Aris Nivorlis
+- Artem Kislovskiy
+- Cyril Bitterich
+- Daksh P. Jain
+- Ege Akman
+- George Zisopoulos
+- Nicolás Demarchi
+- Jakub Červinka
+- Liberta Gani
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Panagiotis Kyrillos
+- Piotr Gnus
+- Saurav Pandey
+- Sebastiaan Zeeff
+- Shekhar Koirala
+- Theofanis Petkos
+
+
+# Sponsors Team
+
+All the sponsorship contacts, logistics & the exhibits are managed by the small team:
+
+- Angel Ramboi
+- Artem Kislovskiy
+- Artur Czepiel
+- Konstantin Ignatov
+- Naa Ashiorkor Nortey
+
+
+# Accessibility Team
+
+This wonderful team worked on making EuroPython 2024 accessible for everyone by setting up Quiet Rooms, Low stimulation room, Childcare and having a Neurodiversity box amongst so many other things.
+
+- Cyril Bitterich
```

Comment:
I think  Alan Dooley  should be listed here. If nothing else, the issues he pointed out and patiently explained has shifted our org to think and approach web accessibility for the better.